src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -41,3 +42,7 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
 export async function createStaticWebAppAdvanced(context: IActionContext, node?: SubscriptionTreeItem): Promise<StaticWebAppTreeItem> {
     return await createStaticWebApp({ ...context, advancedCreation: true }, node);
 }
+
+export async function createStaticWebAppFromLocalProject(context: IActionContext, projectPath: string): Promise<StaticWebAppTreeItem> {
```

Comment:
I don't think that we can assume that the user wants to publish to a new repo when creating a new static web app.

For instance, I can see a scenario where I have a GitHub project, clone it down, work on it a bit, push it up, and then create a new SWA.

I think if we detect a remote, we could assume that the user wants to use that repository.  If there isn't an origin, then we can assume publish to a new repo.